test/cloudnativeproxy/cloudnative_test.go

```diff
@@ -0,0 +1,38 @@
+//go:build e2e
```

Comment:
instead of having `cloudnative` `cloudnativeistio` and `cloudnativeproxy` it should be `cloudnative` `cloudnative/istio` and `cloudnative/proxy`. Different packages seem reasonable due to the requirement of having istio or cillium installed, but having prefixes instead of using packages seems off

Reply:
I did the refactoring for the proxy package, for the istio package I'm not sure if it should be done here, as there have to be at least all paths adapted to the new location. What's your opinion?

Reply:
lets move on in this PR and create a follow up for the istio package

Reply:
I would be fine with that